packages/remix/src/ssr/utils.ts

```diff
@@ -75,3 +66,46 @@ export function assertObject(val: any, error?: string): asserts val is Record<st
     throw new Error(error || '');
   }
 }
+
+/**
+ * @internal
+ */
+export const throwInterstitialJsonResponse = (opts: { frontendApi: string; errorReason: string | undefined }) => {
+  throw json(
+    wrapWithClerkState({
+      __clerk_ssr_interstitial: true,
+      __frontendApi: opts.frontendApi,
+      __lastAuthResult: opts.errorReason,
```

Comment:
❓ 
What do you think about changing it to `__errorReason` to reduce the cognitive load on going through the flow ?

Reply:
Valid point! I think we should go with something like `__lastAuthResult` or similar to be consistent with what we do in NextJS